source/common/secret/secret_manager_impl.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <unordered_map>
+
+#include "envoy/secret/secret.h"
+#include "envoy/secret/secret_manager.h"
+
+#include "common/common/logger.h"
+
+namespace Envoy {
+namespace Secret {
+
+typedef std::unordered_map<std::string, SecretSharedPtr> SecretSharedPtrMap;
```

Comment:
This typedef is unnecessary and/or can be moved private within the impl class.

Reply:
Moved in to the private